tests/playwright/ui-tests/trusts/academies/free-school-meals-page.spec.ts

```diff
@@ -0,0 +1,14 @@
+import { test } from '@playwright/test'
+import { FakeTestData } from '../../../fake-data/fake-test-data'
+import { AcademiesFreeSchoolMealsPage } from '../../../page-object-model/trust/academies/free-school-meals-page'
+
+test.describe('Academies in trust details page', () => {
+  test('user should see the right information about a trust', async ({ page }) => {
+    const freeSchoolMealsPage = new AcademiesFreeSchoolMealsPage(page, new FakeTestData())
+
+    await freeSchoolMealsPage.goTo()
```

Comment:
We should explicitly go to a trust that we know has multiple academies in it to be sure that we're seeing a table containing data

Reply:
Ignore me - I see we already do this in the base academies page!